lib/internal/fs/streams.js

```diff
@@ -395,6 +396,17 @@ WriteStream.prototype.close = function(cb) {
 // There is no shutdown() for files.
 WriteStream.prototype.destroySoon = WriteStream.prototype.end;
 
+WriteStream.prototype._flush = function(cb) {
```

Comment:
Fwiw, the naming is unfortunate here, but please be aware that the stream `_flush()` method does something very different than what the idea for `fsync()` calls is; it’s called just after the last chunk of data has been written, and doesn’t have anything to do with actually flushing data to the underlying resource (it’s not a great name, definitely).